packages/toolkit/src/components/card-model/CardModel.tsx

```diff
@@ -0,0 +1,95 @@
+"use client";
+
+import Image from "next/image";
+import { Menu } from "./Menu";
+import { Tags } from "./Tags";
+import { Stats } from "./Stats";
+import {
+  Model,
+  Nullable,
+  sendAmplitudeData,
+  toastInstillError,
+  useAmplitudeCtx,
+  useDeleteModel,
+} from "../../lib";
+import React from "react";
+import {
+  useToast,
+  getModelHardwareToolkit,
+  getModelRegionToolkit,
+} from "@instill-ai/design-system";
+import { useParams } from "next/navigation";
+
+export type CardModelProps = {
+  model: Model;
+  accessToken: Nullable<string>;
+  onDelete: () => void;
+};
+
+export const CardModel = (props: CardModelProps) => {
+  const { model, accessToken, onDelete } = props;
+  const params = useParams();
+  const entity = params.entity ? String(params.entity) : null;
```

Comment:
@orangecoloured We don't directly interpret entity like this since there will be entity for user and organization. That is the reason we rely on useAppEntity to handle this stuff 🦤